src/GitHub.VisualStudio/UI/Views/Controls/RepositoryCloneControl.xaml.cs

```diff
@@ -60,24 +63,60 @@ public RepositoryCloneControl()
             };
         }
 
-        static ListCollectionView CreateRepositoryListCollectionView(IEnumerable<IRepositoryModel> repositories)
+        ListCollectionView CreateRepositoryListCollectionView(IEnumerable<IRepositoryModel> repositories)
         {
             var view = new ListCollectionView((IList)repositories);
             Debug.Assert(view.GroupDescriptions != null, "view.GroupDescriptions is null");
-            view.GroupDescriptions.Add(new RepositoryGroupDescription());
+            view.GroupDescriptions.Add(new RepositoryGroupDescription(this));
             return view;
         }
 
         class RepositoryGroupDescription : GroupDescription
         {
+            RepositoryCloneControl owner;
+
+            public RepositoryGroupDescription(RepositoryCloneControl owner)
+            {
+                this.owner = owner;
+            }
+
             public override object GroupNameFromItem(object item, int level, System.Globalization.CultureInfo culture)
             {
-                return ((IRepositoryModel)item).Owner.Login;
+                var name = ((IRepositoryModel)item).Owner.Login;
```

Comment:
Ah, good point. As you'll learn, I'm a big fan of assertions when it comes to situations like this because in reviewing code, it's hard to tell the difference from a potential null ref or something we're aware of as being never null.